pytest.ini

```diff
@@ -1,9 +1,12 @@
 # This file is part of Flask-IIIF
-# Copyright (C) 2014 CERN.
+# Copyright (C) 2014, 2015 CERN.
 #
 # Flask-IIIF is free software; you can redistribute it and/or modify
 # it under the terms of the Revised BSD License; see LICENSE file for
 # more details.
 
 [pytest]
-addopts = --clearcache --pep8 --ignore=docs --doctest-modules --cov=flask_iiif --cov-report=term-missing tests flask_iiif
+addopts = --pep8 --ignore=docs --cov=flask_iiif --cov-report=term-missing
+pep8ignore =
+        flask_iiif/* E501
```

Comment:
Well do you really need it in the code? I understand the tests but the code should not have too long lines.